.ci/static-checks.sh

```diff
@@ -271,6 +272,22 @@ check_versions()
 	fi
 }
 
+check_labels()
+{
+	# Since this script is called from another repositories directory,
+	# ensure the utility is built before the script below (which uses it) is run.
+	(cd "${tests_dir}" && make github-labels)
+
+	tmp=$(mktemp)
+
+	info "Checking labels for repo ${repo} using temporary combined database ${tmp}"
+
+	bash -f "${tests_dir}/cmd/github-labels/github-labels.sh" "generate" "${repo}" "${tmp}"
```

Comment:
nitpick - the script is `set -e`, so if this fails, then we'll leave the tmpfile around. We could trap hook its deletion. picky, I know.

Reply:
That's by design - the `info` call specifically mentions the temporary file so that if the script fails, you can inspect it to see wtf caused the problem. The file is tiny so I don't think we'll `ENOSPC` the CI systems ;)